examples/cookbook/design/themes/lib/theme.dart

```diff
@@ -16,9 +54,14 @@ void theme(BuildContext context) {
 
   // #docregion ThemeCopyWith
   Theme(
-    // Find and extend the parent theme using `copyWith`. See the next
-    // section for more info on `Theme.of`.
-    data: Theme.of(context).copyWith(splashColor: Colors.yellow),
+    // Find and extend the parent theme using `copyWith`.
+    // To learn more, check out the next
+    // section on `Theme.of`.
+    data: Theme.of(context)
+        .copyWith(colorScheme: ColorScheme.fromSeed(seedColor: Colors.pink)
+            // TRY THIS: Change the seedColor to "Colors.red" or
```

Comment:
Nit: Move this comment to the line before 61. Comments usually refer to the current line or the following line, not the previous line.

Reply:
**reply:** Usually, I would agree. I want this to come after as it is optional and I want the reader to see the default before another option is presented.

Reply:
In the examples, this pattern usually does have the comment come before the thing that could be changed.  People often run the example before inspecting the code carefully, so in practice they will often be reading the comment after they've run it once anyhow.